resources/assets/sass/components/_masthead.scss

```diff
@@ -0,0 +1,159 @@
+// Masthead
+//
+// Default masthead
+//
+// Markup:
+//<div class="c-masthead">
+//    <h1 class="c-masthead__title">Kuhu sa täna tahaksid minna?</h1>
+//    <div class="c-masthead__wrap">
+//        <div class="c-masthead__logo">
+//            <div class="c-logo"></div>
+//        </div>
+//    </div>
+//</div>
+//
+// default   - Default masthead.
+//
+// Styleguide 8
+
+// Destination masthead
+//
+// Destination masthead
+//
+// Markup:
+//<div class="c-masthead">
+//    <div class="c-masthead__wrap">
+//        <div class="c-masthead__destination-nav m-previous">
+//            <a href="#" class="c-link m-yellow">&lsaquo; Kreeka</a>
+//        </div>
+//        <div class="c-masthead__destination-nav m-next">
+//            <a href="#" class="c-link m-yellow">Itaalia &rsaquo;</a>
+//        </div>
+//    </div>
+//    <div class="c-masthead__heading">
+//        <div class="c-masthead__heading-logo"><div class="c-logo m-small"></div></div>
+//        <h1 class="c-masthead__title">Malta</h1>
+//        <h2 class="c-masthead__subtitle"><a href="#" class="c-masthead__subtitle-link">Euroopa &rsaquo;</a></h2>
+//    </div>
+//</div>
+//
+// default   - Destination masthead.
+//
+// Styleguide 8.1
+
+.c-masthead {
+
+	position: relative;
+	width: 100%;
+	height: 240px;
+	//background-image: url('#{$path-images}/cover-bg-default.jpg');
+	background-repeat: no-repeat;
+	background-position: top center;
+	background-size: cover;
+
+	@include respond-to(desktop) {
+
+		height: 384px;
+	}
+
+	&:after {
+
+		content: '';
+		position: absolute;
+		top: 0;
+		left: 0;
+		width: 100%;
+		height: 100%;
+		background: rgba(#333333, .4);
```

Comment:
Color literals like `#333333` should only be used in variable declarations; they should be referred to via variable everywhere else.<br>`.4` should be written with a leading zero as `0.4`